crates/bevy_asset/src/asset_server.rs

```diff
@@ -572,6 +572,35 @@ mod test {
         )
     }
 
+    #[test]
+    fn missing_asset_loader_error_messages() {
+        assert_eq!(
+            format!(
+                "{}",
+                AssetServerError::MissingAssetLoader { extensions: vec![] }
```

Comment:
You can use `foo.to_string()` instead of `format!("{}", foo)`…

Reply:
Fixed!